src/serviceconnector-passwordless/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.0.1
+++++++
```

Comment:
Better to clearly describe what improvements this change did.

Reply:
updated